### PR TITLE
fix: exporter timeout equal to default prometheus timeout

### PR DESCRIPTION
**Pull request by henri9813**
Fixes: #40 

Motivation: Prometheus default timeout is 10s, and the exporter too !.

So, we currently need to increase the Prometheus timeout to maintain the 10s of the exporter.

This change permit to integrate the exporter to prometheus with a "zero-touch" configuration and avoid overriding prometheus timeout by setting a lower timeout for the exporter.

By the way, a timeout of 10s is huge when we are talking avec eventstore which is really fast.

### CODE REVIEW

**Comment by marcinbudny** [comment]
> By the way, a timeout of 10s is huge when we are talking avec eventstore which is really fast.

With default config and under normal circumstances, this may be true. However there are some options on the exporter (e.g. option to collect stream stats, subscription stats) that may cause the scrape operation to be slow. Additionally if ES is encountering some issues, the responses could also be slower - and this is precisely the moment where metrics reported by the exporter should be available. Perhaps a proper solution to this would be to run the scrape async to requests from Prometheus. Then when Prometheus calls, we would return latest known stats.

Anyway, that would be a larger change and this PR is about adjustment of the default timeout. Let's make the default value 8s instead of 5s. 2s should be plenty for Prometheus to learn about the scrape timeout.

File: README.md [hunk 1]
```diff
@@ -62,13 +62,13 @@ Other versions may also work but are not tested.
 The exporter can be configured with command line arguments, environment variables and a configuration file. For the details on how to format the configuration file, visit [namsral/flag](https://github.com/namsral/flag) repo.
 
 | Flag                           | ENV variable                 | Default                 | Meaning                                                                                                                                 |
-| ------------------------------ | ---------------------------- | ----------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (marcinbudny) [inline README.md:65]: I think this change is not needed.

File: internal/config/config.go [hunk 2]
```diff
@@ -33,7 +33,7 @@ func Load(args []string, suppressOutput bool) (*Config, error) {
 	fs.StringVar(&config.EventStoreUser, "eventstore-user", "", "EventStore User")
 	fs.StringVar(&config.EventStorePassword, "eventstore-password", "", "EventStore Password")
 	fs.UintVar(&config.Port, "port", 9448, "Port to expose scraping endpoint on")
-	fs.DurationVar(&config.Timeout, "timeout", time.Second*10, "Timeout for the scrape operation")
```
Comment (marcinbudny) [inline internal/config/config.go:36]: Let's make it 8s.

**Comment by henri9813** [comment]
Hello,

I applied yours suggestions :)

Best regards

**Pull request merged**